build/AzurePipelinesTemplates/MUX-CreateNugetPackage-Job.yml

```diff
@@ -60,6 +60,13 @@ jobs:
         sourceFolder: '$(Build.SourcesDirectory)\Artifacts\drop'
         Contents: '**\Microsoft.UI.Xaml*.appx'
 
+    # Copy the CBS packages into the same location in the artifact drop to be republished at the end of the job.
```

Comment:
What are these CBS packages that are referred to in this change?

Reply:
CBS change is in master branch, and I cherry-picked to feature/token-experiment.
Here is the original PR
https://github.com/microsoft/microsoft-ui-xaml/pull/3520

Reply:
If you rebase or merge your branch, these changes won't show up as part of the PR.

Reply:
still waiting for #3734